src/components/SideBar.js

```diff
@@ -423,6 +423,7 @@ const SideBar = () => {
       zIndex={1998}
       padding={{ default: 2, s: 0 }}
       spacing={3}
+      onMouseOver={() => setTimeout(window.focus, 200)}
```

Comment:
Why is there a focus of `window`? Why not `e.target` for example?

Reply:
Because you don't need to put focus on the menu items, you need to focus the app window

Reply:
Is that because the iframe is seen as a separate instance than window (rest of the app)?
ecause if I think about it e.target can be different elements.

So this is probably correct 😄

Reply:
Jep, indeed. You're managing focus between 2 windows.